src/utils/index.js

```diff
@@ -32,3 +32,59 @@ export const throttle = ( delay, fn ) => {
  * Get browser locale / language.
  */
 export const getLanguage = () => ( navigator.language || navigator.browserLanguage || ( navigator.languages || [ 'en-US' ] )[ 0 ] ).toLowerCase();
+
+/**
+ * Array merge function for deepmerge.
+ *
+ * @param {Array} destinationArray
+ * @param {Array} sourceArray
+ */
+export const overwriteMerge = ( destinationArray, sourceArray ) => sourceArray;
+
+/**
+ * Resolves an attribute or metric value and sanitize it.
+ *
+ * @param {mixed} value
+ * @param {Function} sanitizeCallback
+ */
+const prepareData = async ( value, sanitizeCallback ) => {
+	if ( typeof value === 'function' ) {
+		value = await value();
+	}
+	if ( ! Array.isArray( value ) ) {
+		value = [ value ];
+	}
+	return value.map( sanitizeCallback );
+};
+
+const sanitizeAttribute = value => value.toString();
+
+const sanitizeMetric = value => parseFloat( Number( value ) );
```

Comment:
It's pretty obvious what these do but docbloc presence is inconsistent in this file